app/components/listing/StreetViewImage.tsx

```diff
@@ -1,46 +1,28 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import React, { useMemo } from 'react';
 import config from '../../config';
+import { Address } from '../../models';
 
-class StreetViewImage extends React.Component {
-  buildImageUrl() {
-    const { address } = this.props;
-
+export const StreetViewImage = ({ address, size = null }: { address: Address; size?: any }) => {
```

Comment:
Seems like `size` is supposed to be a string, based on looking at the old prop types and the fact that this is getting passed into the inline `style` prop.

Reply:
Richard makes a good point here. Null could be passed to the img width/height props. I'm not sure if it would matter, but perhaps a default CSS value would be better, like "auto" or something? Alternatively, if size is null, then the code could just not add it as a style attr to the img element